components/canvas/Canvas.tsx

```diff
@@ -56,10 +58,28 @@ const Canvas = ({
     new Date("2024-04-24T00:08:00.000000Z").getTime();
 
   let tempNodes: Node<NodeDataFull>[] = [];
-  let tempEdges: Edge[] = apiEdges;
+  let tempEdges: Edge[] = [];
+  apiEdges.map((edge: EdgeDataApi) => {
+    const nodeSource = apiNodes.filter((node) => node.id === edge.source);
+    if (nodeSource[0] && nodeSource[0].type === NodeTypes.choice) {
+      tempEdges.push({
+        ...edge,
+        data: { projectId: edge.projectId },
```

Comment:
We can get the projectId from the custom hook so I don't think this is needed

Reply:
removed